### PR TITLE
feat(icons): mini icons set

### PR DESCRIPTION
**Pull request by jeeyun**
Signed-off-by: Jeeyun Lim <jeeyun.lim@gmail.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #4445 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/clr-core/icon-shapes/collections/core.ts [hunk 1]
```diff
@@ -10,90 +10,114 @@ import { IconAlias, IconShapeTuple } from '../interfaces/icon.interfaces.js';
...
 import { vmBugInverseIcon, vmBugInverseIconName } from '../shapes/vm-bug-inverse.js';
 import { vmBugIcon, vmBugIconName } from '../shapes/vm-bug.js';
 import { warningStandardIcon, warningStandardIconName } from '../shapes/warning-standard.js';
+import { warningMiniIcon, warningMiniIconName } from '../shapes/warning-mini.js';
 
 export const coreCollectionIcons: IconShapeTuple[] = [
   angleIcon,
   angleDoubleIcon,
   arrowIcon,
+  arrowMiniIcon,
```
Comment (mathisscott) [inline src/clr-core/icon-shapes/collections/core.ts:64]: These mini icons should go into their own collection...

File: src/clr-core/icon-shapes/shapes/arrow-mini.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { renderIcon } from '../icon.renderer.js';
+import { IconShapeTuple } from '../interfaces/icon.interfaces.js';
+
+const icon = {
+  outline:
+    '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 36 36"><g id="Layer_3" data-name="Layer 3"><path d="M29.18,13.26,17.92,3,6.63,13.28a2,2,0,0,0-.55,2.33,2,2,0,0,0,3.19.68L16,10.13V30.29a2,2,0,0,0,1.35,2A2,2,0,0,0,20,30.38V10.28l6.57,6a2,2,0,0,0,1.35.52,2,2,0,0,0,1.72-1A2.08,2.08,0,0,0,29.18,13.26Z"/></g></svg>',
```
Comment (mathisscott) [inline src/clr-core/icon-shapes/shapes/arrow-mini.ts:12]: We don't include the SVG wrappers with the icons. See https://github.com/vmware/clarity/blob/master/src/clr-core/icon-shapes/shapes/airplane.ts for an example of how your icon should look. We should only see paths here.

File: src/clr-core/icon-shapes/collections/mini.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import { checkMiniIcon, checkMiniIconName } from '../shapes/check-mini.js';
+import { errorMiniIcon, errorMiniIconName } from '../shapes/error-mini.js';
+import { eventMiniIcon, eventMiniIconName } from '../shapes/event-mini.js';
+import { filterGridCircleMiniIcon, filterGridCircleMiniIconName } from '../shapes/filter-grid-circle-mini.js';
+import { filterGridMiniIcon, filterGridMiniIconName } from '../shapes/filter-grid-mini.js';
+import { infoCircleMiniIcon, infoCircleMiniIconName } from '../shapes/info-circle-mini.js';
+import { timesMiniIcon, timesMiniIconName } from '../shapes/times-mini.js';
+import { warningMiniIcon, warningMiniIconName } from '../shapes/warning-mini.js';
+
+export const miniCollectionIcons: IconShapeTuple[] = [
```
Comment (coryrylan) [inline src/clr-core/icon-shapes/collections/mini.ts:22]: Can we add a new test for this collection to this?

https://github.com/vmware/clarity/blob/master/src/clr-core/icon-shapes/collections/collections.spec.ts

Comment (jeeyun) [reply src/clr-core/icon-shapes/collections/mini.ts:22]: Yes. Thanks for catching that.

**Pull request merged**